versions.lock

```diff
@@ -121,13 +121,13 @@ com.papertrail:profiler:1.0.2 (1 constraints: e21051b8)
 com.squareup:javapoet:1.13.0 (3 constraints: 1c204572)
 com.squareup.okhttp3:logging-interceptor:3.13.1 (1 constraints: b114c190)
 com.squareup.okhttp3:okhttp:3.13.1 (3 constraints: f835e1be)
-com.squareup.okio:okio:1.17.5 (2 constraints: c4113007)
+com.squareup.okio:okio:2.10.0 (2 constraints: b9118505)
```

Comment:
this will likely cause some downstream deps to bump, might require fixing any okio 2.x API breaks (they [hid Base64, ByteStringKt, Platform, and Util code from Java](https://github.com/square/okio/releases/tag/okio-parent-2.0.0))

https://github.com/square/okio/blob/master/CHANGELOG.md